src/kritter/ktabularclient.py

```diff
@@ -0,0 +1,16 @@
+class KtabularClient:
+
+    def __init__(self):
+        pass
+
+    '''
+    Creates a google sheet with the requested data 
+    '''
+    def createGS(self,fileName,USER,data): # data is a panda data frame
+        pass
+
+    '''
+    Gets the spreadsheet's URL and returns it as a string
+    '''
+    def getURL(self):
+        pass
```

Comment:
I'm not a big style guy... but PEP8 suggests using camel case with classes and using lowercase and underscores with variables and functions.  Please apply this to your variable names in GtabularClient, and lets make createGS "create" (don't need the GS) and it should return a reference so we can manipulate it. (Imagine if we have more than 1 sheet we're working with. )  getURL (get_url) should take the reference as an arg.